pep_pre_commit_hooks/core.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import re
+import subprocess
+
+
+def verify_git_email(domain: str) -> None:
+    command = ("git", "config", "--get", "user.email")
+    output = subprocess.check_output(command).decode().strip()
+    if re.search(f".*@{re.escape(domain)}$", output):
+        return
+    else:
+        raise ChildProcessError(
```

Comment:
Yeah, was also wondering. But since there could also be other thigns going wrong here (like invalid email but correct domain), I did not want to be very specific here. Plus this won't really ever be called from Python, but as a CLI program through the hook in interactive use, so I think that should be sufficient.